packages/examples/twitter-verifier/README.md

```diff
@@ -0,0 +1,419 @@
+# ZK Email Verify
```

Comment:
There's a lot of info in here that applies to the broader codebase, not Twitter - can you isolate that out?